morf-testsupport/pom.xml

```diff
@@ -44,5 +44,9 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
     </dependency>
+    <dependency>

```

Comment:
This one is more controversial though.  A dirty trick we could pull is to make `DatabaseInspector.inspect()` a `default` method which uses `ServiceLoader` to locate an implementation on the classpath.  That way you can move `H2DatabaseInspector` to `morf-h2` (test tree) but leave `DatabaseInspector` here.

Reply:
That's not a bad idea.

Reply:
Hmmm, but it's in the `testsupport` project, and I am uncomfortable about moving `H2DatabaseInspector` to the h2 `main` tree.